packages/sdk/src/evm/core/classes/staking.ts

```diff
@@ -0,0 +1,57 @@
+import { BigNumber, BigNumberish, ethers } from "ethers";
+import { Amount } from "../../types";
+import { BaseERC1155, BaseERC20, BaseERC721 } from "../../types/eips";
+import { ContractWrapper } from "./contract-wrapper";
+import { AmountSchema } from "../../../core/schema/shared";
+import { AddressOrEns } from "../../schema";
+import { resolveAddress } from "../../common";
+
+export class Staking {
```

Comment:
this doesnt need to be a class, i would just do helper functions instead of inheriting